website/docs/faqs/Troubleshooting/ide-session-unknown-error.md

```diff
@@ -0,0 +1,20 @@
+---
+title: I'm receiving an 'Your IDE session experienced an unknown error and was terminated. Please contact support'.
+description: "Add a repository when seeing IDE unknown error"
+sidebar_label: 'Receiving unknown error in the IDE'
+id: IDE
+
+---
+
+If you're experiencing the below error when you try to open the cloud IDE, it may be due to a few scenarios but it typically indicates a missing repository.
+
+```shell
+
+Your IDE session experienced an unknown error and was terminated. Please contact support
+
+```
+
+To resolve this, we recommend adding a repository like a [managed repository](/docs/collaborate/git/managed-repository) or your preferred Git account by navigating to Project > Repository and selecting your repository.
```

Comment:
since we don't know if these steps will actually "resolve this" bc there are several reasons a user could run into this error, sugg slightly diff wording. 

something like this or similar: 

also, for dropdown menus, we bold the UI text

```suggestion
You can try to resolve this by adding a repository like a [managed repository](/docs/collaborate/git/managed-repository) or your preferred Git account. To add your Git account, navigate to **Project** > **Repository** and select your repository.
```